devskope/iReporter
# #163142486 Implement general dashboard functionality

**What does this PR do?**
Implementation of a general dashboard accessible to valid, logged in users.

**Description of Task to be completed?**
- View all records created by all users
- Filter records by type and or status
- See the number of records that are
   - drafts
   - under investigation
   - resolved
   - rejected
- View details of a single record.

**How should this be manually tested?**
- Clone this repository
- In your local copy checkout `ft-implement-general-dash-163142486`
- run `npm run devstart`

 Test dashboard functionality by signing up for a new iReporter account. You should be directed to the dashboard upon successful registration

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[163142486](https://www.pivotaltracker.com/story/show/163142486)

Screenshots (if appropriate)

N/A

Questions:

N/A
